docs/missing-docs.md

```diff
@@ -0,0 +1,55 @@
+# Known Missing Glossary / Documentation
```

Comment:
i think you're wrong

Reply:
> i think you're wrong

Why? :thinking:

As I see it, GitHub Issues are used to track issues, problems, bugs, feature requests. :thinking:

Reply:
Yes, and I think you're wrong. Given that we have yet to even move this PR forward in several weeks, for what is quite literally the most minor of documentation updates I've seen us try to add in months, I doubt we could accurately track what is being fixed.

Now, if we get that issue properly created and fleshed out, this list can be removed and documented as "migrated to #issue"

Reply:
my point being that if we separated this by individual issues for each and every one, it'd slam our issue queue and muck it up worse than it already is, and we should worry about working what we can, and if we're so concerned about completing our documentation, then you can create an issue off of the ones you want to fix. alternatively, if you don't like how i documented it, make the issue, and i'll remove it from here.

i'm not the only one who can update it, i'm trying to help because of the sheer slow pace that is the norm right now.

Reply:
for the sake of trying to make some movement on this, i've created #640

Reply:
> Given that we have yet to even move this PR forward in several weeks

Sorry about that.

> my point being that if we separated this by individual issues for each and every one

That was not what I wanted to suggest. I wanted to have a single issue for all the known missing glossary items, which you have in the end created in #640 (thank you very much). 

> you can create an issue off of the ones you want to fix. alternatively, if you don't like how i documented it, make the issue, and i'll remove it from here.

This is very true. I can, but the free time I have had recently (maybe since the start of the year) does not permit me to contribute much recently. Thanks for understanding. :pray: However, I’ll do my best to work on romcal more.